shared/android/app/src/main/java/io/keybase/ossifrage/KBReactPackage.java

```diff
@@ -10,8 +10,8 @@
 import java.util.Iterator;
 import java.util.List;
 
-import io.keybase.ossifrage.components.KBTextInputManager;
-import io.keybase.ossifrage.modules.AppearanceModule;
+//import io.keybase.ossifrage.components.KBTextInputManager;
```

Comment:
we don't need this old workaround for text input
and we don't need our own dark mode code (appearance)